inference-engine/tests/functional/plugin/cpu/single_layer_tests/roi_pooling.cpp

```diff
@@ -0,0 +1,210 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <shared_test_classes/single_layer/roi_pooling.hpp>
+#include "ie_common.h"
+#include "test_utils/cpu_test_utils.hpp"
+#include "utils/bfloat16.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+
+namespace CPULayerTestsDefinitions {
+
+using ROIPoolingCPUTestParamsSet = std::tuple<LayerTestsDefinitions::roiPoolingParamsTuple,
+                                              CPUSpecificParams,
+                                              std::map<std::string, std::string>>;
+
+class ROIPoolingCPULayerTest : public testing::WithParamInterface<ROIPoolingCPUTestParamsSet>,
+                               virtual public LayerTestsUtils::LayerTestsCommon,
+                               public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<ROIPoolingCPUTestParamsSet> obj) {
+        LayerTestsDefinitions::roiPoolingParamsTuple basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::map<std::string, std::string> additionalConfig;
+
+        std::vector<size_t> inputShape;
+        std::vector<size_t> coordsShape;
+        std::vector<size_t> poolShape;
+        float spatial_scale;
+        ngraph::helpers::ROIPoolingTypes pool_method;
+        InferenceEngine::Precision netPrecision;
+        std::string targetDevice;
+
+        std::tie(basicParamsSet, cpuParams, additionalConfig) = obj.param;
+        std::tie(inputShape, coordsShape, poolShape, spatial_scale, pool_method, netPrecision, targetDevice) = basicParamsSet;
```

Comment:
Are we sure we need to tie this parameters, which are not used in this function at all?

Reply:
My bad, fixed